cscs-checks/microbenchmarks/gpu/pointer_chase/pointer_chase.py

```diff
@@ -3,16 +3,18 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
-import reframe.utility.sanity as sn
+import os
+
 import reframe.utility.typecheck as typ
 import reframe as rfm
 
-import os
+import library.microbenchmarks.gpu.pointer_chase as pchase
+import cscslib.microbenchmarks.gpu.hooks as hooks
 
 
 class PchaseGlobal(rfm.RegressionMixin):
```

Comment:
I am really struggling with the name choices... `PchaseGlobal`... Does it make sense to have it more specific?

Reply:
Yeah, not the best.  Let me change this to something better. I'll also change the imports to avoid having the `chase. ...` and `hooks. ...`

Reply:
Changed to `BaseCSCS`